message_definitions/v1.0/csAirLink.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<mavlink>
+  <!-- ClearSky Air-Link contact info:          -->
+  <!-- company URL: https://air-link.space/     -->
+  <!-- email contact: drone@air-link.space      -->
+  <!-- mavlink ID range: 8500 - 8600            -->
```

Comment:
Yeah, you're not stealing our space :-P

Please grab a block of a hundred or so in here: https://github.com/mavlink/mavlink/blob/master/message_definitions/v1.0/all.xml#L42